static/app/views/issueDetails/groupEventCarousel.tsx

```diff
@@ -337,20 +378,26 @@ export function GroupEventCarousel({event, group, projectSlug}: GroupEventCarous
           ]}
         />
         {xlargeViewport && (
-          <Button size={BUTTON_SIZE} onClick={copyLink}>
-            Copy Link
-          </Button>
+          <Tooltip title={t('Copy link to this issue event')}>
+            <Button
+              size={BUTTON_SIZE}
+              onClick={copyLink}
+              icon={<IconLink />}
+              aria-label={t('Copy Link')}
+            />
+          </Tooltip>
         )}
         {xlargeViewport && (
-          <Button
-            size={BUTTON_SIZE}
-            icon={<IconOpen size={BUTTON_ICON_SIZE} />}
-            onClick={downloadJson}
-          >
-            JSON
-          </Button>
+          <Tooltip title={t('View JSON')}>
```

Comment:
You can just pass `tooltip={t('View JSON')}` here instead of wrapping it in the component

Reply:
title={t('View JSON')}